### ISSUE TITLE
applying focal and dice loss functions in densenet 161 incurs errors.

### ISSUE THREAD

**Issue by Elhamahm** [CosmiQ/solaris#349]
Hello, 
For densenet161 model, when I use focal and dice loss functions, then it caused the following bug issue in training step. Any help and advise for solving this issue is appreciated. Thanks

![image](https://user-images.githubusercontent.com/7827229/80225906-f49cd280-8608-11ea-940e-d4ebe92ceef0.png)
